package.json

```diff
@@ -31,7 +31,8 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook --no-dll",
     "lint": "eslint --ext ts --ext tsx --fix src/",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "prisma generate"
```

Comment:
Adding this script to the package.json to ensure that the cached prisma client on Vercel gets re-generated that way changes are picked up and a fresh client is available.

Source: https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-vercel#generate-prisma-during-build